tests/system/src/test/java/test/robot/javafx/embed/swing/SwingNodePlatformExitCrashTest.java

```diff
@@ -25,18 +25,18 @@
 
 package test.robot.javafx.embed.swing;
 
-import javafx.application.Platform;
-import org.junit.Ignore;
-import org.junit.Test;
-import test.util.Util;
-import javax.swing.SwingUtilities;
 import java.lang.reflect.InvocationTargetException;
 import java.util.concurrent.CountDownLatch;
+import javax.swing.SwingUtilities;
+import javafx.application.Platform;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import test.util.Util;
 
 public class SwingNodePlatformExitCrashTest extends SwingNodeBase {
 
     @Test
-    @Ignore("JDK-8190329")
+    @Disabled("JDK-8190329")
```

Comment:
Unrelated to your PR...well this is embarrassing! I just fixed the FX clone of JDK-8190329 a couple weeks ago, and forgot to look to see if there was an existing test. I'll file a follow-up bug for this.

Reply:
I filed [JDK-8340442](https://bugs.openjdk.org/browse/JDK-8340442).